pkg/didcomm/packer/anoncrypt/pack.go

```diff
@@ -101,6 +107,26 @@ func (p *Packer) Pack(payload, _ []byte, recipientsPubKeys [][]byte) ([]byte, er
 	return []byte(s), nil
 }
 
+func jweTypesFromOpts(opts []packer.Opts) (string, string) {
+	pOpts := packer.NewOpt()
+
+	for _, opt := range opts {
+		opt(pOpts)
+	}
+
+	typ := pOpts.TypHeader()
+	if typ == "" {
+		typ = packer.EncodingTypeV2
+	}
+
+	cty := pOpts.CTYHeader()
+	if cty == "" {
+		cty = packer.ContentEncodingTypeV2
```

Comment:
This is incorrect - default cannot be DIDComm V2 media types since they are not part of the core group in AIP 2.0

Reply:
ContentEncodingType (`cty`) is passed to the packers, there's no more default, this code is outdated